src/twisted/internet/test/test_endpoints.py

```diff
@@ -1324,7 +1324,7 @@ def createServerEndpoint(self, reactor, factory, **listenArgs):
         address = IPv4Address("TCP", "0.0.0.0", 0)
 
         if listenArgs is None:
-            listenArgs = {}
```

Comment:
In order for `listenArgs` to be `None` here, it would have to be passed in as a keyword argument itself:

```python
>>> def createServerEndpoint(self, reactor, factory, **listenArgs):
...     pass
... 
>>> createServerEndpoint(None, None, None, listenArgs=None)
>>> 
```
 
It's a weird thing to do, and I don't see any uses of that in Twisted itself, but uh… ok.